template/template.go

```diff
@@ -39,6 +39,24 @@ func (t *ServiceMethod) extractFirstTag() (string, error) {
 	return tagName, nil
 }
 
+func (t *ServiceMethod) extractAllTags() ([]string, error) {
+	opts, ok := t.Method.Option("grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation").(*options.Operation)
+	if !ok {
+		return nil, fmt.Errorf("grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation from method %s", t.Method.Name)
+	}
+	optsTags := opts.Tags
+	var tagNames []string
+	for _, tagName := range optsTags {
+		tagNames = append(tagNames, tagName)
+	}
+
+	if len(tagNames) == 0 {
+		return nil, fmt.Errorf("couldn't find any tags for method %s", t.Method.Name)
```

Comment:
maybe just log the error here and return an empty array?

Reply:
I thought returning an empty array was a bad practice in go.

Reply:
yeah okay return nil. dont think we need to return error